### PR TITLE
Bugfix pr to 1.11

### PR DESCRIPTION
**Pull request by cltnschlosser**

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/colony/Colony.java [hunk 1]
```diff
@@ -468,16 +468,21 @@ public void onServerTick(@NotNull final TickEvent.ServerTickEvent event)
...
+
         //  Recompute subscribers every frame (for now)
         //  Subscribers = Owners + Players within (double working town hall range)
         @NotNull final Set<EntityPlayerMP> oldSubscribers = subscribers;
         subscribers = new HashSet<>();
 
         // Add owners
         subscribers.addAll(
-          this.getWorld().getMinecraftServer().getPlayerList().getPlayers()
+          world.getMinecraftServer().getPlayerList().getPlayers()
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/colony/Colony.java:485]: this did cause the nullpointer exception?

Comment (cltnschlosser) [reply src/main/java/com/minecolonies/coremod/colony/Colony.java:485]: world was null or MinecraftServer reference was null.

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/colony/Colony.java:485]: kk

**Comment by Raycoms** [comment]
:shipit: I'd say

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/548/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**